src/bci_build/package/__init__.py

```diff
@@ -602,11 +567,21 @@ def build_version(self) -> str | None:
 
     @property
     def eula(self) -> str:
-        return self._image_properties.eula
+        """EULA covering this image. Is ``sle-eula`` for LTSS codestreams,
+        otherwise ``sle-bci``.
+
+        """
+        return "sle-eula" if self.os_version.is_ltss else "sle-bci"
```

Comment:
this is missing the "is_sles and is application type container" check. because those are also "sle-eula"